GENERATOR.md

```diff
@@ -0,0 +1,119 @@
+# Generated Prow jobs
+
+## Presubmits
+
+### Tests
+
+For each
+[test](https://github.com/openshift/ci-operator/blob/master/CONFIGURATION.md#tests)
+specified in the configuration file, generate one presubmit running ci-operator
+to build this test target.
+
+```yaml
```

Comment:
The whole YAML is not really useful. Maybe outline the:

 - job name
 - context name
 - rerun command